src/modals/examples/modal-overlay.js

```diff
@@ -0,0 +1,179 @@
+ * and Patternfly styling. Use cases for Modal Overlays can vary a great deal, but some examples include the following:
+ * <ul>
+ * <li>To remind or prompt users
+ * <li>To load dialogs that require user input before advancing
+ * <li>To load tasks which require a user’s full attention, such as stepping through a wizard flow
+ * <li>To present important information or warnings
+ * </ul>
+ *
+ * @param {boolean} isOpen Flag indicating that the modal should be opened
+ * @param {function} onClose Function to call when the modal is closed
+ * @param {string} modalId Id of the modal
+ * @param {string} modalTitle The title of the modal displayed in the header
+ * @param {string=} titleId Id of the title. "modalTitle" by default
+ * @param {boolean=} hideCloseIcon Flag indicating that the modal should hide the 'x' close icon.
+ * @param {boolean=} backdropClose Flag indicating that the modal should close if user clicks background. False by default
+ * @param {boolean=} isForm Flag indicating that the modal body will contain a form which should be valid before an action button is enabled
+ * @param {string} modalBodyTemplate Path to html template for modal body
+ * @param {object=} modalBodyScope Object containing the scope for the modalBodyTemplate
+ * @param {array} actionButtons array of button objects. Each button can have the following properties:<br/>
+ * <ul style='list-style-type: none'>
+ * <li>.label        - the text to display on the button
+ * <li>.class        - (optional) classes to add to the button
+ * <li>.actionFn     - (optional) use defined function to call when the button is clicked
+ * <li>.isDisabled   - (optional) boolean true if the button should be disabled by default
+ * <li>.isCancel     - (optional) boolean true is the button should cancel and dismiss the modal
+ * </ul>
+ *
+ * @example
+ <example module="patternfly.modals">
+
+ <file name="index.html">
+ <div ng-controller="DemoModalOverlayCtrl">
+    <button ng-click="open()" class="btn btn-default">Launch Modal Overlay</button>
+    <pf-modal-overlay is-open="isOpen"
+          on-close="onClose()"
+          modal-id="modalId"
+          modal-title="modalTitle"
+          modal-body-template="template"
+          modal-body-scope="formScope"
+          action-buttons="actionButtons"
+          is-form="isForm">
+    </pf-modal-overlay>
+
+   <button ng-click="open2()" class="btn btn-default">Launch Second Modal Overlay</button>
+   <pf-modal-overlay is-open="isOpen2"
+         on-close="onClose2()"
+         modal-id="modalId2"
+         modal-title="modalTitle2"
+         hide-close-icon="hideCloseIcon2"
+         backdrop-close="backdropClose"
+         title-id="titleId2"
+         modal-body-template="template2"
+         action-buttons="actionButtons2">
+    </pf-modal-overlay>
+   <div>
+      <label class="actions-label">Actions: </label>
+   </div>
+   <div>
+      <textarea rows="3">{{actionsText}}</textarea>
```

Comment:
In order to get the Actions textarea to display correctly.  Add `class="col-md-12"` here